pyspecProcScripts/__init__.py

```diff
@@ -22,12 +22,10 @@
 from .integral_w_error import integral_w_errors, active_propagation
 from .apod_matched_filter import apod_matched_filter
 from .envelope import L2G, fit_envelope
-from .first_level.QESR_rescale import QESR_scalefactor
```

Comment:
this PR hasn't been merged into master yet so at this point this line shouldn't be here